ci/docs/publish.md

```
@@ -0,0 +1,15 @@
+# Publish a stemcell.
+## Verify stemcell builds
+Whenever a USN (Ubuntu Security Notice) is published, a ticket is automaticly created in the community-stemcell board (https://github.com/orgs/cloudfoundry/projects/4).
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)